DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/BaseConversionToSpeckle.cs

```diff
@@ -5,14 +5,19 @@
 namespace Speckle.Converters.RevitShared.ToSpeckle;
 
 // POC: maybe but could be generic abstract Converters.Common?
+// or maybe it's not actually doing very much now and can come out
 public abstract class BaseConversionToSpeckle<THost, TSpeckle>
   : IHostObjectToSpeckleConversion,
+    // POC: why do we need to do this for each base conversion?
     IRawConversion<THost, TSpeckle>
   where TSpeckle : Base
 {
   public Base Convert(object target)
   {
     var result = RawConvert((THost)target);
+
+    // POC: unless I am going bonkers, we've constrained TSpeckle to Base
+    // so it should always BE base?
     if (result is not Base @base)
```

Comment:
Oh yeah... not sure how we'll ever get here 🤔

Reply:
We have the POC comment but the whole class seems a bit bonkers now